x11-libs/gtk+/gtk+-1.2.10-r13.ebuild

```diff
@@ -49,6 +49,8 @@ src_prepare() {
 }
 
 multilib_src_configure() {
+	append-cflags -std=gnu89
```

Comment:
This appends it twice or more when the func is called multiple times. You either need to define full src_configure() and call multilib-minimal_src_configure there, or you can resort to src_prepare in this case.